src/middlewared/middlewared/plugins/chart_releases_linux/helm.py

```diff
@@ -17,6 +18,8 @@ class Config:
     @private
     def helm_action(self, chart_release, chart_path, config, tn_action):
         args = ['-f']
+        CLEANUPREGEX = re.compile(r'^Error: .+[)]: ')
```

Comment:
Please take the declaration of this variable to line 12 and then add a couple of new lines for flake8. Also we have an unspoken norm for naming scheme like `RE_CLEANUP` or something like that - which is basically prefixing `RE_` with regex compiles done.

Reply:
@sonicaj The condition is there, because it is only working (correctly) for a single(!) specific type of errors.
It is not, and will not be, working (correctly) for others.

You're basically pushing data it's not, validated to be, compatible with, through this regex.
Are you 100% sure that you want to be doing that?

Reply:
The other issue (constant defining at top of the file) has been fixed as `RE_ERRCLEANUP`

Reply:
> @sonicaj The condition is there, because it is only working (correctly) for a single(!) specific type of errors. It is not, and will not be, working (correctly) for others.
> 
> You're basically pushing data it's not, validated to be, compatible with, through this regex. Are you 100% sure that you want to be doing that?

@Ornias1993 I don't follow ? We only have 3 possible values for tn actions i.e install/upgrade/update and for helm that just translates to upgrade/install which means that there is no other case ?

Reply:
As it's technically a general Helm-Call function, I personally find it nicer to assume it will be used for more than those two commands.

But As you want it changed i'll remove the if :)

Reply:
Resolved :)